### ISSUE TITLE
Add choose-your-own-avatar functionality to Hide Avatars

### ISSUE THREAD

**Issue by dlmarquis** [new-xkit/XKit#769]

**Comment by coderpatsy** [comment]
Wasn't this left out because it made it too easy to fake posts?

**Comment by dlmarquis** [comment]
No, that was a Sasukimous thing. Hide Avatars currently dims the Nx icon;
we can just leave that in/make it more noticable when we push these changes.

**Comment by dlmarquis** [comment]
I've added the requisite code to my local copy, but I keep forgetting about it when I'm at home. Someone kick me in the face to get this tested and made into a PR.
